src/CMakeLists.txt

```diff
@@ -222,9 +222,11 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
   set(APP_NAME ${APP_NAME}-dbg)
 endif()
 if(CMAKE_APP_TYPE STREQUAL "BMDK")
-    add_subdirectory(apps/bm_devkit/${APP})
+	add_subdirectory(apps/bm_devkit/${APP})
 elseif(CMAKE_APP_TYPE STREQUAL "bristleback")
-    add_subdirectory(apps/bristleback_apps/${APP})
+	add_subdirectory(apps/bristleback_apps/${APP})
+elseif(CMAKE_APP_TYPE STREQUAL "rs232_expander")
```

Comment:
add new app type... not a huge fan of the name but I think that "rs232" would be too generic since you can technically to rs232 on both the devkit and bristleback,

Reply:
Good call.